### ISSUE TITLE
NPM Module Confusion

### ISSUE THREAD

**Issue by shellscape** [milligram/milligram-cssnext#2]
Not a typical issue, but something I feel could be clarified for those who are inquisitive...

The README states that using `npm install milligram-cssnext` is the preferred method to install milligram-cssnext, however the module at https://www.npmjs.com/package/milligram-cssnext is linked to @steven-klein's fork of this repo, and not a module published under milligram's account.

What's the story behind the inconsistency between the project's repo and the npm module's repo entry and owner?

**Comment by steven-klein** [comment]
@shellscape I agree this is inconsistent and confusing.  I believe it was just an unintended consequence of starting the fork and initially publishing it to NPM before it became an official part of the milligram project.  

milligram-cssnext is overdue for an update.  I'll work on making sure everything is consistent with the next pull request.

**Comment by alanonthegit** [comment]
Installing through bower is also broken. Any updates?

**Comment by steven-klein** [comment]
Hey guys, sorry this has been sitting open for so long.  I took on a new position and haven't had much time to get back to this.  I think the reality is this project probably just needs to go away.  At the end of the day milligram only has 6 variables and it's proving to be too cumbersome to keep up to date and change a lot of syntax just so it can be consumed by postcss-cssnext.  It's likely to introduce more errors than it's worth, the demand really isn't there.  I'm not sure what @cjpatoilo has in mind for the future of milligram.  I know there have been requests to allow it to be more heavily customized through variables and other means, but as it stands now it just needs to be piped through node-sass, or have a few additional postcss plugins added before cssnext, if a user want's to customize the variables in their asset pipeline.

@ar-stackrox To answer your question.  We didn't have a definitive release plan for this project, which is on me, but It wasn't published to bower, and probably won't be at this point.  Best bet is to add it via npm or yarn.

**Comment by cjpatoilo** [comment]
Hey @steven-klein 

The features of CSS Next are not just the variables but if this feature is extremely relevant to maintaining this repository, implement that and keep at least one stable version. We can not accept a repository that does not work.

I think so: if there are people coming after information about the product is because there is demand. Are you interested and willing to maintain this repository? If your answer is yes, do it. If not, could you give me the accesses of the accounts so that I can give support?

My info to transfer:

User: cjpatoilo
Email: cjpatoilo@gmail.com

**Comment by steven-klein** [comment]
@cjpatoilo This repository does implement postcss-cssnext's vairables.  That was the reason it was created since it simplifies using milligram in a purely postcss workflow.  I'm just feeling like for most users running milligram through node-sass before importing it into their postcss-cssnext workflow (which is what I will personally start doing) is a better option than trying to maintain multiple versions of this product.

The current state of this repo is working fine it would just need to be deployed to the bower registry.  It is a little behind the base milligram repo unfortunately, tracking the changes was just too difficult with this being a separate repository.  

In any event, I've made you an owner of milligram-cssnext at the npm registry, so you can publish a latest version and I'll remove myself as an owner completing the transfer.
